test/test_battle.h

```diff
@@ -65,8 +65,9 @@
  * single turn. MOVE causes the player to use Stun Spore and adds the
  * move to the Pokémon's moveset if an explicit Moves was not specified.
  * Pokémon that are not mentioned in a TURN use Celebrate.
- * The test runner attempts to rig the RNG so that the first move used
- * in a turn does not miss and activates its secondary effects (if any).
+ * The test runner rigs the RNG so that unless otherwise specified,
+ * moves always hit, never critical hit, always activate their secondary
+ * effects, and always roll the same damage modifier.
```

Comment:
It is not super clear to me now how I would specify that I want either randomness (idk if thats a requirement) or a specific mechanic to still work, e.g. how would I test against crits? Do I need to specify a `PASSES_RANDOMLY` with a tag?